src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -75,42 +92,68 @@ private async Task CreateBlocksAsync()
             {
                 try
                 {
-                    // Don't mine in IBD.
-                    if (this.ibdState.IsInitialBlockDownload() && !MineDuringIBD)
+                    // Don't mine in IBD in case we are connected to any node.
+                    if (this.ibdState.IsInitialBlockDownload() && this.connectionManager.ConnectedPeers.Any())
```

Comment:
If we're in IBD and connected to no one, it will start mining. Is that the intention?

Reply:
Yes

Reply:
don't we want to be out of IBD first?

Reply:
We can't if we are not connected to anyone. 

So let's say we ran the node (we are the only miner on network), then we shutted it down for 2 days and then started up. Now we will never mine because we are in IBD. 

Also it's ok to mine even if we are on a disconnected part of the network- we will reorg to better chain anyway always (there is no MaxReoirg property so it doesn't matter for how long we are disconnected)

Reply:
Good point.